README.md

```diff
@@ -0,0 +1,93 @@
+[![Lint](https://github.com/Mean-Finance/subsidies/actions/workflows/lint.yml/badge.svg?branch=main)](https://github.com/Mean-Finance/subsidies/actions/workflows/lint.yml)
```

Comment:
Should we rename the repository/package to 'ongoing airdrops' or 'campaigns'?

Reply:
To be honest, I think we should go for `campaigns`.